app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError)

```

Comment:
fix import brackets according to the best practicies 
![image](https://user-images.githubusercontent.com/103201010/219699817-620fc52e-cf00-4e03-83a8-003098bca6b4.png)